org.lflang/src/org/lflang/ast/FormattingUtils.java

```diff
@@ -48,10 +51,24 @@ public class FormattingUtils {
      * {@code lineLength}.
      */
     public static String render(EObject object, int lineLength) {
+        return render(object, lineLength, inferTarget(object), false);
+    }
+
+    /** Return a function that renders AST nodes for the given target. */
+    public static Function<EObject, String> renderer(TargetDecl targetDecl) {
+        return object -> render(object, DEFAULT_LINE_LENGTH, Target.fromDecl(targetDecl), true);
+    }
+
+    /**
+     * Return a String representation of {@code object}, with lines wrapped at
+     * {@code lineLength}, with the assumption that the target language is
+     * {@code target}.
+     */
+    public static String render(EObject object, int lineLength, Target target, boolean codeMapTags) {
         MalleableString ms = ToLf.instance.doSwitch(object);
-        String singleLineCommentPrefix = getSingleLineCommentPrefix(object);
+        String singleLineCommentPrefix = target == Target.Python ? "#" : "//";
```

Comment:
This will not work for all targets. I couldn't find it, but I thought we had a method somewhere that returns the comment prefix

Reply:
I agree, this isn't right. Will attempt a fix.

Reply:
Fixed.